tutorials/text_processing/WFST_Tutorial.ipynb

```diff
@@ -0,0 +1,8368 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "jLpm4mufAfUz"
+   },
+   "source": [
+    "However this would make our graph [non-deterministic](https://en.wikipedia.org/wiki/Nondeterministic_algorithm) - have multiple possibilities for termination. Now an input of \"cents\" could have the outcome of `100` or `100s` when only one is correct. "
```

Comment:
havING? also i think it cannot produce `100s`

Reply:
I think I mentioned that it's assuming closure over all characters, or should that be more explicitly mentioned?

Reply:
Nevermind, I changed it so it doesn't require the closure assumption.